### PR TITLE
Add sink tests for builtins

### PR DESCRIPTION
**Pull request by bitterpanda63**

### CODE REVIEW

File: aikido_firewall/sinks/subprocess.py [hunk 1]
```diff
@@ -17,18 +16,18 @@ def generate_aikido_function(op, former_func):
     """
 
     def aikido_new_func(*args, op=op, former_func=former_func, **kwargs):
-        logger.debug("Wrapper - `subprocess` on %s() function", op)
         shell_enabled = kwargs.get("shell")
 
-        if isinstance(args[0], list):
+        command = None
+        if len(args) != 0 and hasattr(args[0], "__iter__"):
```
Comment (willem-delbare) [inline aikido_firewall/sinks/subprocess.py:22]: lets comment the intent here in these complex statements

File: aikido_firewall/sinks/os.py [hunk 2]
```diff
@@ -37,8 +37,14 @@ def generate_aikido_function(op, former_func):
     """
 
     def aikido_new_func(*args, op=op, former_func=former_func, **kwargs):
-        #  args[0] : filename
-        run_vulnerability_scan(kind="path_traversal", op=f"os.{op}", args=(args[0],))
+        if len(args) > 0 and isinstance(args[0], str):  #  args[0] : filename
+            vulns.run_vulnerability_scan(
+                kind="path_traversal", op=f"os.{op}", args=(args[0],)
+            )
+        if len(args) > 1 and isinstance(args[1], str):  # args[1] : Could be dst folder
```
Comment (willem-delbare) [inline aikido_firewall/sinks/os.py:44]: do we have unit tests covering this? what fn? lets comment

Comment (bitterpanda63) [reply aikido_firewall/sinks/os.py:44]: Fairly certain os_test covers this

File: aikido_firewall/sinks/subprocess.py (hunk 1 above)
Comment (bitterpanda63) [reply aikido_firewall/sinks/subprocess.py:22]: Done

**Pull request merged**